### DISCUSSION TITLE
Take advantage of z coordinate to allow overlap of different units

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1030] [Ideas]
Hi:

Currently for different units the physics doesn't allow them to be overlap if we set them collidable 
<img width="960" alt="2BE277ED-5D1E-4456-97D2-C9C042219348" src="https://user-images.githubusercontent.com/5525436/116323636-04895c80-a7f1-11eb-8977-ad09c02f20b8.png">
is it possible to use z coordinate that allow them to overlap?
like this 
```java
var unit = spawn("tank", x: 100, y: 0, z: -1);
var unit2 = spawn("tank", x: 100, y: 0, z: -2);
```
then unit 1 & 2 could overlap in x: 100 & y:0

Thanks

**Comment by chengenzhao** [comment]
hmmm……
seems like i know how to do it
using collidable entity without physics

**Reply by AlmasB** [reply]
Yes, only use `PhysicsComponent` if you need physics simulations, where entities are treated as rigid bodies. In other cases, just a collidable entity should be fine.
